SingularityRunnerBase/src/main/java/com/hubspot/singularity/runner/base/shared/JsonObjectFileHelper.java

```diff
@@ -28,15 +28,17 @@ public JsonObjectFileHelper(ObjectMapper objectMapper) {
 
     log.info("Reading {}", file);
 
-    byte[] bytes = Files.readAllBytes(file);
+    byte[] bytes = new byte[0];
 
-    log.trace("Read {} bytes from {} in {}", bytes.length, file, JavaUtils.duration(start));
+    try {
+      bytes = Files.readAllBytes(file);
 
-    if (bytes.length == 0) {
-      return Optional.absent();
-    }
+      log.trace("Read {} bytes from {} in {}", bytes.length, file, JavaUtils.duration(start));
+
+      if (bytes.length == 0) {
+       return Optional.absent();
+      }
```

Comment:
instead of these checks, i'd just explicitly handle `NoSuchFileException` in the try-catch